generate-release/src/release_notes.rs

```diff
@@ -67,11 +67,18 @@ pub fn generate_release_notes(
         contributors.remove(&author);
 
         // Make sure the author is always the first in the list
-        let mut authors = vec![author];
-        authors.extend(contributors);
+        let authors = vec![author];
```

Comment:
does this still need to be a vec? Are there situations where you can have multiple authors per PR if you don't include the contributors?

Reply:
I don't think so; I'll simplify the code ASAP.